src/h3lib/include/constants.h

```diff
@@ -80,5 +80,7 @@
 /** H3 index modes */
 #define H3_HEXAGON_MODE 1
 #define H3_UNIEDGE_MODE 2
+#define H3_EDGE_MODE 3
+#define H3_VERTEX_MODE 4
```

Comment:
Please add the new mode to the specification in docs

Reply:
Updated the Indexing docs, reformatting, clarifying a bit, and adding the new mode.